inst/tests/tests.Rraw

```diff
@@ -10720,7 +10722,7 @@ test(1743.08, sapply(fread("a,b,c\n2017-01-01,1,1+3i", colClasses=c("Date", "int
 test(1743.09, sapply(fread("a,b,c\n2017-01-01,1,1+3i", colClasses=c("Date", "integer", "complex")), class), c(a="Date", b="integer", c="complex"))
 test(1743.10, sapply(fread("a,b,c,d\n2017-01-01,1,1+3i,05", colClasses=c("Date", "integer", "complex", NA)), class), c(a="Date",b="integer",c="complex",d="integer"))
 test(1743.11, sapply(fread("a,b,c,d\n2017-01-01,1,1+3i,05", colClasses=c("Date", "integer", "complex", "raw")), class), c(a="Date",b="integer",c="complex",d="raw"))
-test(1743.12, x = vapply(fread("a,b\n2015-01-01,2015-01-01", colClasses = c(NA, "IDate")), inherits, what = "IDate", FUN.VALUE = logical(1)), y = c(a=FALSE, b=TRUE))
+test(1743.12, x = vapply(fread("a,b\n1+3i,2015-01-01", colClasses = c(NA, "IDate")), inherits, what = "IDate", FUN.VALUE = logical(1)), y = c(a=FALSE, b=TRUE))
```

Comment:
I'm not quite clear the purpose of this test. But before, `colClasses=NA` led the first column type to be guessed; now it is correctly guessed as `IDate`, but the test expected `!inherits(., 'IDate')`

Reply:
but you changed fread input, no? maybe that change should not be here but in another PR, no?

Reply:
Hmm shall I just remove the test?

Maybe the test is: `as(x, 'IDate')` gets applied because it was in `colClasses`, but the other column stays `character`. In which case a better test would be to re-tailor the test toward `complex` instead of `IDate`...

Reply:
In eaf2d57 I reinstated the original test but changed the result, kept your new test and added another.  So 1743.12 becomes 3: 1743.12[1-3]. Think that covers everything here.